R/estimateDiversity.R

```diff
@@ -586,22 +591,22 @@ setMethod("estimateFaith", signature = c(x="TreeSummarizedExperiment", tree="mis
     return(faiths)
 }
 
-.calc_log_modulo_skewness <- function(mat, quantile = 0.5, num_of_classes = 50, ...){
+.calc_log_modulo_skewness <- function(mat, quantile = 0.5, nclasses = 50, ...){
```

Comment:
nclasses was chose to be the name because we have also parameter ntop which can also be found from scater package